app/ts/components/pages/InterceptorAccess.tsx

```diff
@@ -1,20 +1,58 @@
 import { useState, useEffect } from 'preact/hooks'
 import { ActiveAddress, BigAddress, WebsiteOriginText } from '../subcomponents/address.js'
 import { AddNewAddress } from './AddNewAddress.js'
-import { AddressInfoEntry, AddressBookEntry, AddingNewAddressType, RenameAddressCallBack, AddressInfo, Website, WebsiteSocket, SignerName } from '../../utils/user-interface-types.js'
-import { ExternalPopupMessage } from '../../utils/interceptor-messages.js'
+import { AddressInfoEntry, AddressBookEntry, AddingNewAddressType, RenameAddressCallBack, Website } from '../../utils/user-interface-types.js'
+import { ExternalPopupMessage, PendingAccessRequest, PendingAccessRequestArray } from '../../utils/interceptor-messages.js'
 import { sendPopupMessageToBackgroundPage } from '../../background/backgroundUtils.js'
 import Hint from '../subcomponents/Hint.js'
 import { convertNumberToCharacterRepresentationIfSmallEnough, tryFocusingTab } from '../ui-utils.js'
 import { ChangeActiveAddress } from './ChangeActiveAddress.js'
-import { DinoSays } from '../subcomponents/DinoSays.js'
+import { DinoSays, DinoSaysNotification } from '../subcomponents/DinoSays.js'
 import { getPrettySignerName } from '../subcomponents/signers.js'
 
+const HALF_HEADER_HEIGHT = 48 / 2
+
+
+function Title({ icon, title} : {icon: string | undefined, title: string}) {
+	return <p style = 'font-weight: 700; line-height: 48px'>
+		{ icon === undefined
+			? <></>
+			: <img src = { icon } style = 'width: 48px; height: 48px; vertical-align: bottom; margin-right: 10px;'/>
+		}
+		{ title }
+	</p>
+}
+
+function AccessRequestHeader(website: Website) {
+	return <header class = 'card-header' style = 'height: 40px'>
+		<div class = 'card-header-icon noselect nopointer' style = 'width: 100%;'>
+			<WebsiteOriginText { ...website } />
+		</div>
+	</header>
+}
+
+type UnderTransactionsParams = {
+	reversedPendingAccessRequestArray: PendingAccessRequestArray
+}
+
+function UnderAccesses(param: UnderTransactionsParams) {
+	const nTx = param.reversedPendingAccessRequestArray.length
+	return <div style = {`position: relative; top: ${ nTx * -HALF_HEADER_HEIGHT }px;`}>
```

Comment:
consider using objects as style properties here
```suggestion
	return <div style = {{ position: 'relative', top: nTx * -HALF_HEADER_HEIGHT }}>
```

Reply:
added issue about this, I have been planning to move into this styling but I havent gotten aroudn it